knowledge-base/setup-checklist/kubeone/kubeone-baremetal-requirements.md

```diff
@@ -0,0 +1,38 @@
+# Additional Requirements for Bare Metal Setups
+
+## Access to BareMetal Infrastructure
+
+1. Access by SSH via VPN or JumpHost to the dedicated bare metal host
+2. The provided ssh user needs to have sudo rights to set up kubernetes
+3. List of IP addresses, to provision the target environment 
+
+**NOTE:** Dynamic provisioning is currently in Development. Alternative approaches are to use a virtualized [kubevirt](https://github.com/kubermatic/machine-controller/blob/master/docs/cloud-provider.md#kubevirt) based setup at Kubermatic KKP. 
+
+## Kubernetes Cluster / Network separation
+
+The separation and multitenancy of KubeOne clusters is highly dependent on the provided network and user management of the bare metal Infrastructure. Due to the individuality of such setups it's recommended to create a dedicated concept per installation together with Kubermatic engineering team. Please provide at least one separate network CIDR and technical user for the management components and each expected tenant.
+
+As an alternative, for protecting the dedicated kubernetes cluster could be improved by applying so called "Host Protection Network Policies" by default trough Calico, see [https://docs.projectcalico.org/security/protect-hosts](https://docs.projectcalico.org/security/protect-hosts)
+
+In-cluster multitenancy is manageable by dedicated RBAC configuration, see [Kubernetes Documentation > API Access Control > Using RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) what is enabled by default with every KubeOne Cluster. For applying a core set of RBAC configuration, the KubeOne addon could manage the settings, see [KubeOne Documentation > Advanced > Addons](https://docs.kubermatic.com/kubeone/master/advanced/addons/).
+
+## Routable virtual IPs (for metalLB)
+
+To set up Kubermatic behind [MetalLB](https://metallb.universe.tf/), we need a few routable, not distributed IP address. This could be sliced into one CIDR. The virtual IPs should be routed to the target network, but not used for machines. 
```

Comment:
The phrase "not distributed IP address." might be confusing. I'd rather say "not managed by DHCP" or similar.